src/java.base/share/classes/module-info.java

```diff
@@ -381,7 +383,6 @@
         java.prefs;
     exports sun.util.resources to
         jdk.localedata;
-
```

Comment:
Nit: I'd suggest to preserve the empty line (since there's no other change near).

Reply:
fixed, thanks.